validations/src/validations.rs

```diff
@@ -2430,20 +2442,25 @@ mod tests {
     fn test_block_reward() {
         // 1 wit = 10^9 nanowits, block_reward returns nanowits
         let wit = 1_000_000_000;
-
-        assert_eq!(block_reward(0), 250 * wit);
-        assert_eq!(block_reward(1), 250 * wit);
-        assert_eq!(block_reward(1_749_999), 250 * wit);
-        assert_eq!(block_reward(1_750_000), 125 * wit);
-        assert_eq!(block_reward(3_499_999), 125 * wit);
-        assert_eq!(block_reward(3_500_000), (62.5 * wit as f64).floor() as u64);
-        assert_eq!(block_reward(1_750_000 * 36), 3);
-        assert_eq!(block_reward(1_750_000 * 37), 1);
-        assert_eq!(block_reward(1_750_000 * 38), 0);
-        assert_eq!(block_reward(1_750_000 * 63), 0);
-        assert_eq!(block_reward(1_750_000 * 64), 0);
-        assert_eq!(block_reward(1_750_000 * 65), 0);
-        assert_eq!(block_reward(1_750_000 * 100), 0);
+        let ibr = INITIAL_BLOCK_REWARD;
+        let hp = HALVING_PERIOD;
```

Comment:
You can use a closure to avoid passing the same arguments all the time:

```suggestion
        let reward = |epoch| block_reward(epoch, INITIAL_BLOCK_REWARD, HALVING_PERIOD);
```